splinterd/src/main.rs

```diff
@@ -501,6 +501,14 @@ fn main() {
                 .takes_value(true),
         );
 
+    #[cfg(feature = "scabbard-storage-config")]
+    let app = app.arg(
+        Arg::with_name("scabbard-state")
+            .long("scabbard-state")
+            .long_help("Where scabbard should store state, options are 'lmdb', and 'database'")
```

Comment:
You can add .possible_values(&["lmdb", "database"]) here to have clap validate the value passed.